app/assets/javascripts/discourse/models/login-method.js.es6

```diff
@@ -79,7 +79,7 @@ const LoginMethod = Ember.Object.extend({
 
 let methods;
 
-export function findAll(siteSettings, capabilities, isMobileDevice) {
```

Comment:
We do not need the extra parameters anymore. This simplifies the code at all the places where `findAll` is being used.